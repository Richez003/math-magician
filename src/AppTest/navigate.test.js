import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navigate from '../page/navigate'

it('NavBar component should perform correctly on initial run', () => {
    const tree = renderer.create(<Router><Navigate /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
  });