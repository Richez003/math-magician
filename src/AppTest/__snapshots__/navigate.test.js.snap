// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component should perform correctly on initial run 1`] = `
<header
  className="head"
>
  <h1>
    Math Magician
  </h1>
  <nav>
    <a
      href="/"
      onClick={[Function]}
    >
      HomePage|
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      href="/quote"
      onClick={[Function]}
    >
      |Quotes
    </a>
  </nav>
</header>
`;

exports[`renders correctly 1`] = `
<section
  className="home-wrapper"
>
  <h2>
    Welcome To Our Page
  </h2>
  <div
    className="body-wrapper"
  >
    <p>
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita! Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita!!
    </p>
    <p>
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita! Lorem, ipsum dolor sit amet consectetur adipisicing elit. Omnis, distinctio. Quae eos fugiat corrupti ex fugit, repudiandae deleniti illo quisquam, rem veritatis cupiditate maiores itaque eligendi possimus quaerat quis expedita!!
    </p>
  </div>
</section>
`;
